src/harness/unittests/session.ts

```diff
@@ -422,9 +422,10 @@ namespace ts.server {
             handle(msg: protocol.Message): void {
                 if (msg.type === "response") {
                     const response = <protocol.Response>msg;
-                    if (response.request_seq in this.callbacks) {
-                        this.callbacks[response.request_seq](response);
-                        delete this.callbacks[response.request_seq];
+                    const handler = this.callbacks.get(response.request_seq);

```

Comment:
Either `callbacks` should be a sparse array, or we should coerce `response.request_seq` to a string.